app/services/hyrax/thumbnail_path_service.rb

```diff
@@ -24,7 +24,7 @@ def call(object)
         def fetch_thumbnail(object)
           return object if object.thumbnail_id == object.id
           Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: object.thumbnail_id)
-        rescue Valkyrie::Persistence::ObjectNotFoundError
+        rescue Hyrax::ObjectNotFoundError
```

Comment:
@lsitu - I'm wondering how come you replace Valkyrie::Persistence::ObjectNotFoundError with Hyrax::ObjectNotFoundError.  From the ticket, it mentions that use it to replace ActiveFedora::ObjectNotFoundError.  Thanks.

Reply:
@VivianChu I believe that's what `Hyrax::ObjectNotFoundError` is designed to handle, which will wrap up any such `ObjectNotFoundError` in Hyrax.

Reply:
@lsitu - Thanks for the clarification.  LGTM 👍